tests/functional/test_reflector.py

```diff
@@ -219,6 +219,37 @@ def verify_blob_completed(blob, blob_size):
         d.addCallback(lambda _: verify_data_on_reflector())
         return d
 
+    def test_blob_reflector_v1(self):
+        def verify_data_on_reflector():
+            check_blob_ds = []
+            for blob_hash, blob_size in self.expected_blobs:
+                check_blob_ds.append(verify_have_blob(blob_hash, blob_size))
+            return defer.DeferredList(check_blob_ds)
+
+        def verify_have_blob(blob_hash, blob_size):
+            d = self.server_blob_manager.get_blob(blob_hash, True)
+            d.addCallback(lambda blob: verify_blob_completed(blob, blob_size))
+            return d
+
+        def send_to_server(blobs_to_send):
```

Comment:
we need to be consistent with our variable name to distinguish between whether an object is a blob_hash string, a HashBlob, etc.  

I think `blobs_to_send` should be `blob_hashes`